### PR TITLE
Bump up the event queue's maxLimit

### PR DESCRIPTION
**Pull request by Praveenrajmani**
## Description
Bumping up the maxLimit of the events queue

## Motivation and Context
To persist even more events

## How to test this PR?
Try setting the queue_limit > 100000

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [x] Documentation needed
- [ ] Unit tests needed
- [ ] Functional tests needed (If yes, add [mint](https://github.com/minio/mint) PR # here: )

### CODE REVIEW

**Comment by starinacool** [comment]
I still suggest a million as a limit or remove a limit completely. On our deployment we have more then 200K PUTs a day.

Setting a 100 000 as a default and dropping a limit is a correct solution from my point of view.

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-xl.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-azure.sh | :x: [more...](#77b8a95c-0f7d-4cc0-b3e4-8f8960e01a49) |
| mint-gateway-nas.sh | :x: [more...](#4dd2b99e-c0da-4164-be1b-4a948c6088f1) |
<a name="4dd2b99e-c0da-4164-be1b-4a948c6088f1"></a>
### 9380-84692d5/mint-gateway-nas.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-dev3.minio.io:32760
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 16e12b6ff61b:/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 0 seconds
(2/15) Running aws-sdk-java tests ... done in 2 seconds
(3/15) Running aws-sdk-php tests ... done in 41 seconds
(4/15) Running aws-sdk-ruby tests ... done in 1 seconds
(5/15) Running awscli tests ... done in 2 minutes and 8 seconds
(6/15) Running healthcheck tests ... done in 0 seconds
(7/15) Running mc tests ... done in 38 seconds
(8/15) Running minio-dotnet tests ... done in 29 seconds
(9/15) Running minio-go tests ... FAILED in 30 seconds
{}

Executed 8 out of 15 tests successfully.
```
<a name="77b8a95c-0f7d-4cc0-b3e4-8f8960e01a49"></a>
### 9380-84692d5/mint-gateway-azure.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-c3.minio.io:32648
ACCESS_KEY:           minioazure
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp a97cc8787281:/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 7 seconds
(2/15) Running aws-sdk-java tests ... done in 2 seconds
(3/15) Running aws-sdk-php tests ... done in 2 minutes and 2 seconds
(4/15) Running aws-sdk-ruby tests ... done in 15 seconds
(5/15) Running awscli tests ... FAILED in 56 seconds
{
  "name": "awscli",
  "duration": 4760,
  "function": "aws --endpoint-url http://minio-c3.minio.io:32648 s3api head-object --bucket awscli-mint-test-bucket-661 --key datafile-1-kB-copy\n",
  "status": "FAIL",
  "error": "StorageClass was not applied"
}

Executed 4 out of 15 tests successfully.
```
Deleting image on docker hub
Deleting image locally

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-xl.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-azure.sh | :x: [more...](#abedd16a-b3c7-4194-8ecd-71d7ab72a792) |
| mint-gateway-nas.sh | :x: [more...](#7b6c07d8-94a8-46d6-8589-6b0618a093ac) |
<a name="7b6c07d8-94a8-46d6-8589-6b0618a093ac"></a>
### 9380-f60fb66/mint-gateway-nas.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-c3.minio.io:32693
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 26933a717cc1:/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 0 seconds
(2/15) Running aws-sdk-java tests ... done in 2 seconds
(3/15) Running aws-sdk-php tests ... done in 40 seconds
(4/15) Running aws-sdk-ruby tests ... done in 2 seconds
(5/15) Running awscli tests ... done in 2 minutes and 8 seconds
(6/15) Running healthcheck tests ... done in 0 seconds
(7/15) Running mc tests ... done in 27 seconds
(8/15) Running minio-dotnet tests ... done in 30 seconds
(9/15) Running minio-go tests ... FAILED in 40 seconds
{}

Executed 8 out of 15 tests successfully.
```
<a name="abedd16a-b3c7-4194-8ecd-71d7ab72a792"></a>
### 9380-f60fb66/mint-gateway-azure.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-dev7.minio.io:30714
ACCESS_KEY:           minioazure
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 8f07d9586f44:/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 8 seconds
(2/15) Running aws-sdk-java tests ... done in 1 seconds
(3/15) Running aws-sdk-php tests ... done in 2 minutes and 8 seconds
(4/15) Running aws-sdk-ruby tests ... done in 34 seconds
(5/15) Running awscli tests ... FAILED in 53 seconds
{
  "name": "awscli",
  "duration": 4867,
  "function": "aws --endpoint-url http://minio-dev7.minio.io:30714 s3api head-object --bucket awscli-mint-test-bucket-22841 --key datafile-1-kB-copy\n",
  "status": "FAIL",
  "error": "StorageClass was not applied"
}

Executed 4 out of 15 tests successfully.
```
Deleting image on docker hub
Deleting image locally

File: pkg/event/target/queuestore.go [hunk 1]
```diff
@@ -30,7 +30,7 @@ import (
 )
 
 const (
-	maxLimit = 10000 // Max store limit.
```
Comment (harshavardhana) [inline pkg/event/target/queuestore.go:33]: using the same variable for `maxLimit` looks confusing @Praveenrajmani we should say perhaps defaultLimit?

**Review by harshavardhana — CHANGES_REQUESTED** [review]
also you need to update the actual help in `cmd/config/notify/help.go`

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-xl.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-azure.sh | :x: [more...](#afe8f754-71d6-4cf4-92e6-6e95fa7ab454) |
| mint-gateway-nas.sh | :x: [more...](#ecb8e10c-b67d-4ab3-812b-b340101d043c) |
<a name="ecb8e10c-b67d-4ab3-812b-b340101d043c"></a>
### 9380-5cc66dc/mint-gateway-nas.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-dev6.minio.io:30475
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 7f49ba4dc339:/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 0 seconds
(2/15) Running aws-sdk-java tests ... done in 1 seconds
(3/15) Running aws-sdk-php tests ... done in 41 seconds
(4/15) Running aws-sdk-ruby tests ... done in 2 seconds
(5/15) Running awscli tests ... done in 2 minutes and 5 seconds
(6/15) Running healthcheck tests ... done in 0 seconds
(7/15) Running mc tests ... done in 24 seconds
(8/15) Running minio-dotnet tests ... done in 28 seconds
(9/15) Running minio-go tests ... FAILED in 38 seconds
{}

Executed 8 out of 15 tests successfully.
```
<a name="afe8f754-71d6-4cf4-92e6-6e95fa7ab454"></a>
### 9380-5cc66dc/mint-gateway-azure.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-dev7.minio.io:32577
ACCESS_KEY:           minioazure
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 187ec263920f:/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 7 seconds
(2/15) Running aws-sdk-java tests ... done in 2 seconds
(3/15) Running aws-sdk-php tests ... done in 2 minutes and 4 seconds
(4/15) Running aws-sdk-ruby tests ... done in 15 seconds
(5/15) Running awscli tests ... FAILED in 55 seconds
{
  "name": "awscli",
  "duration": 4585,
  "function": "aws --endpoint-url http://minio-dev7.minio.io:32577 s3api head-object --bucket awscli-mint-test-bucket-31873 --key datafile-1-kB-copy\n",
  "status": "FAIL",
  "error": "StorageClass was not applied"
}

Executed 4 out of 15 tests successfully.
```
Deleting image on docker hub
Deleting image locally

**Review by harshavardhana — APPROVED** [review]

**Review by BigUstad — APPROVED** [review]
LGTM

**Pull request merged**